src/base.css

```diff
@@ -573,6 +710,11 @@
 		border-style: none;
 	}
 
+	img, svg {
+		/* This isn't color-scheme aware, on purpose. */
+		background: white;
+	}
```

Comment:
This is a breaking change, whereas the rest of it isn't. It's possible that an image has been authored to be transparent and isn't paying enough attention to the distance between the figure and the image edge to look good. I'm not sure this is safe to apply to existing documents.